build.sbt

```diff
@@ -64,6 +65,12 @@ lazy val api = (projectMatrix in file("modules") / "api")
   .settings(commonSettings, apiSettings)
   .jvmPlatform(scalaVersions = versions.fullCrossScalaVersions)
 
+/** Java-only project that contains interfaces for a language-agnostic mutator API
+  */
+lazy val mutatorApi = (project in file("modules") / "mutatorApi")
+  .settings(mutatorApiSettings)
+// .dependsOn(testkit.jvm(versions.scala3) % Test)
```

Comment:
I've added this and commented _if_ we want to add any tests. We could also add junit and write them in Java, but this seemed less work. I think for now we don't need any tests